config/kip500-combined.properties

```diff
@@ -125,4 +125,4 @@ log.segment.bytes=1073741824
 
 # The interval at which log segments are checked to see if they can be deleted according
 # to the retention policies
-log.retention.check.interval.ms=300000
+log.retention.check.interval.ms=300000
```

Comment:
let's remove this whitespace-only change

Reply:
Super weird. My commit history is not tracking this change. Let me re-merge and check